dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/ClassloadingTest.java

```diff
@@ -55,17 +60,30 @@ public static void setUp() throws Exception {
         // Build test app with just a test servlet (i.e. no JSF usage)
         ShrinkHelper.defaultApp(server, NO_JSF_APP, "jsf.container.nojsf.web");
 
-        // Location of Mojarra jars
-        String mojarraLibraryLocation  = "publish/files/mojarra/";
-
-        if(JakartaEE9Action.isActive()){
-          mojarraLibraryLocation  = "publish/files/mojarra30/";
+        
+        String mojarraLibraryLocation;
+        WebArchive mojarraAppWar;
+
+        // Multiple checks due to the managed bean refactoring for faces 4.0
+        if (JakartaEE10Action.isActive()) {
```

Comment:
I realize I'm a bit inconsistent with my use of `JakartaEE10Action.isActive()` and isEE10, isEE9, ...etc. when you compare classes.